modules/xmpp/JingleSessionPC.js

```diff
@@ -1009,7 +1009,7 @@ export default class JingleSessionPC extends JingleSession {
             sid: this.sid
         });
 
-        new SDP(offerSdp).toJingle(
+        new SDP(offerSdp, this.isP2P).toJingle(
```

Comment:
Do we foresee any extra options to this constructor?

Reply:
Nope, this is all we would need and that is why I didn't create an options parameter for this.

Reply:
Ack.